.github/workflows/build_and_test.yml

```
@@ -660,50 +660,12 @@ jobs:
             })
 
   trigger-e2e-tests:
+    if: ${{ !github.event.pull_request.draft || contains( github.event.pull_request.labels.*.name, 'run-e2e-test-in-draft') }}
```

- Will this job be triggered in branches (when merged into `main` and into `release`)?
- Leftovers from the previous label renaming:
```suggestion
    if: ${{ !github.event.pull_request.draft || contains( github.event.pull_request.labels.*.name, 'run-e2e-tests-in-draft') }}
```